components/psst/browser/core/DEPS

```diff
@@ -1,4 +1,5 @@
 include_rules = [
   "+crypto",
+  "+content/public/test",
```

Comment:
you con't include `content` deps in core, that defeats the entire purpose of `content` vs `core`

Reply:
see https://github.com/brave/brave-core/pull/29044/files#r2101138542